examples/angular/src/app/app.component.ts

```diff
@@ -54,7 +54,7 @@ export class AppComponent implements OnInit {
         setupMeteorWallet(),
         setupCoin98Wallet(),
         setupWalletConnect({
-          projectId: "c4f79cc...",
+          projectId: "88fb772e055e421763933a2908e4229a",
```

Comment:
Can you please remove your WalletConnect **projectId** from both examples?

We do not add a specific projectId because anyone who uses this project can and needs to have their own WalletConnect projectId,.
Other developers might make the mistake and rely on this projectId which you probably will delete or change later on the WalletConnect dashboard.